easybuild/easyconfigs/j/jupyterlab/jupyterlab-1.2.16-CrayGNU-20.06-cuda-batchspawner.eb

```diff
@@ -0,0 +1,295 @@
+# @author: robinson (omlins and hvictor for IJulia)
+
+easyblock = 'PythonBundle'
+
+name = 'jupyterlab'
+version = '1.2.16'
+versionsuffix = '-cuda-batchspawner'
+
+_pyminver = '8'
```

Comment:
```suggestion
```
not used anywhere ?

Reply:
Let's wait for Tim to see if he wants to keep / remove it.

Reply:
remove, looks stale